website/blog/2020-05-18-rn4mupdadates.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "React Native for macOS"
+author: Kiki Saintonge
+authorURL: http://twitter.com/KikiSaintonge
+description: "We're on React Native core version 0.62 with new community modules, native module support across multiple platforms, and of course our official public release of React Native for macOS."
+is_blog: true
+published_date: May 17, 2020
+---
+
+We're excited to announce our first preview release aligning with [React Native 0.62](https://reactnative.dev/blog/2020/03/26/version-0.62)!
+
+<!--truncate-->
+
+As a preview release, we will try our best not to make breaking changes, but still have a few bumps to sort out before we're ready for release. You can now start trying out the [0.62-preview of React Native for Windows](https://www.npmjs.com/package/react-native-windows/v/0.62.0-preview.6)!
+
+## What's New?
+
+- New Features from React Native 0.62, like [`Appearance`](https://reactnative.dev/docs/appearance) and [`useColorScheme()`](https://reactnative.dev/docs/usecolorscheme) hooks
+- Visual Studio V142 build tools are now used
+-	Better RTL support
+-	New native module APIs (`REACT_INIT` and `REACT_FUNCTION`)
+-	Improved diagnostics in command line tools
+-	Initial ARM64 support
+-	Experimental support for binary distribution of React Native Windows
+-	Bug fixes and reliability improvements
+
+## React Native for macOS
+
+![react-native-mac](assets/eloy_rn4m_preview_full.png)
+
+One of our biggest announcements recently is our addition and full support of the React Native for macOS extension! We are super excited to be developing React Native support for macOS along side our Windows effort.
+
+Although still a fork, we have an official [npm package out for React Native for macOS](https://www.npmjs.com/package/react-native-macos) and a [repo available](https://github.com/microsoft/react-native-macos) to the public to follow and contribute to - we hope to see some really cool React Native apps out there in the future that can now truly run everywhere.

```

Comment:
Remove "Although still a fork". That's an implementation detail I don't want to call attention to anymore.

Reply:
removed.